app/src/services/api/generated/api.generated.ts

```diff
@@ -1462,14 +1462,14 @@ export type PatchedOwnerRequest = {
   name?: string;
   credential?: string;
 };
-export type SeverityEnum = "fatal" | "error" | "warning" | "information";
 export type OperationOutcomeIssue = {
-  severity: SeverityEnum;
+  severity: string;
```

Comment:
You definitely shouldn't have those changes in your PR. I guess the merge you made was not from the latest commit of `v4` ?